goformation_test.go

```diff
@@ -1289,4 +1315,60 @@ var _ = Describe("Goformation", func() {
 			})
 		})
 	})
+
+	Context("with a SAM template", func() {
+
+		Context("described as Go structs", func() {
+			template := cloudformation.NewTemplate()
+			transform := "AWS::Serverless-2016-10-31"
+			transformStruct := &cloudformation.Transform{
+				String: &transform,
+			}
+			template.Transform = transformStruct
+
+			codeUri := "hello-world/"
+			template.Resources["HelloWorldFunction"] = &serverless.Function{
+				CodeUri: &serverless.Function_CodeUri{
+					String: &codeUri,
+				},
+				Handler: "hello-world",
+				Runtime: "go1.x",
+			}
+
+			globals := cloudformation.Globals{}
+			globals["Function"] = &global.Function{
+				Timeout: 123,
+			}
+			template.Globals = &globals
```

Comment:
Is this test actually running? I can't see it failing in the PR, but `template.Globals` field doesn't exist in your PR (I don't see any modifications to `cloudformation/template.go`).

Reply:
Ack, I did edit this file but I thought it was a codegenned file and I didn't commit it. Will take a second look through my local diff...